### ISSUE TITLE
fix(select): Cant bind to property of object array without an optionsTemplate

### ISSUE THREAD

**Issue by trevordaniels** [edcarroll/ng2-semantic-ui#157]
If I try to bind to a specific property on an object array without specifying a template, it throws the following error even when using labelField and valueField

```
<sui-select #select [formControl]="formControl" [options]="[{'description':1},{'description':2}]" 
[valueField]="description" [labelField]="description" >
     <sui-select-option *ngFor="let option of select.filteredOptions" [value]="option"></sui-select-option>
</sui-select>
```

```
ERROR TypeError: text.replace is not a function at SearchService.highlightMatches 
```

I cant get select to work in plunkr at the moment so no plunkr for this.

**Comment by edcarroll** [comment]
I've had a look and this is twofold, firstly you're binding `valueField` and `labelField` to the value of `description`, not the value `description` (remove the square brackets).

However, this has raised the fact that passing a non-string field will cause an error, so I've updated `labelField` to always return a string, so if you don't change this code you'll end up with lots of `[object Object]` in your select which will highlight the issue.

The hotfix release containing these changes will be out soon.

**Comment by trevordaniels** [comment]
I realised this after I submitted the issue, removed the brackets and was
still getting the same error.

**Comment by edcarroll** [comment]
Makes sense, since the 2nd thing was that it's a non-string field so `replace` wouldn't work. Once I've sorted out the checkbox form stuff I'll get this release out :)

**Comment by trevordaniels** [comment]
I see what you mean. Yes, removing square brackets and specifying a string
field works as expected. Not specifying a label field also throws the same
error.

>

**Comment by edcarroll** [comment]
This is now fixed in `0.9.2`, the value of the property specified in `labelField` is automatically cast to a string before running any functions. (This also works for when nothing is specified)
